src/main/resources/db/migration/V1__create_movie_table.sql

```diff
@@ -1,7 +1,7 @@
-create table movie (
-    id bigserial primary key,
-    title varchar(100),
-    description varchar(500),
-    release_year int,
-    rating float(2)
+CREATE TABLE movie (
+    id bigserial PRIMARY KEY,
+    title VARCHAR(100),
+    description VARCHAR(500),
+    release_year INT,
+    rating FLOAT(2)
```

Comment:
You made some symbols uppercase, did you make sure, that this change don't break migration because I have a feeling that checksum might be different for this migration.

Reply:
Reverted changes.